src/utils/Api.ts

```diff
@@ -0,0 +1,72 @@
+const baseURL = '';
+const baseHeaders = {
+  'Content-Type': 'application/json',
+};
+
+const checkResponse = (res: Response) => {
+  if (res.ok) return res.json();
+  return res.json().then((err) => Promise.reject(err));
+};
+
+interface IcheckResponse {
```

Comment:
```suggestion
interface IСheckResponse {
```
и TCheckSuccess там ниже

Но лучше не делать префиксы, разницы в смысле нет. Экономия в одной буковке будет небольшой. А вот будет большой экономия в том, что не надо будет думать как назвать какую-нибудь сущность. Нужно будет добавлять ей префикс или нет, а если добавлять, то какой.
Родные типы HTML, реакта и тайпскрипта без префиксов